rootfs/builder/build.sh

```diff
@@ -206,4 +206,5 @@ if [[ "$slug_file" != "-" ]]; then
 				curl -0 -s -o /dev/null -X PUT -T $slug_file "$put_url"
 			fi
 		fi
+	fi
```

Comment:
I think this is an indentation issue. Notice that there is 4 `if` levels here and 5 `fi` levels.